src/data/repositories/consts/DiseaseOutbreakConstants.ts

```diff
@@ -33,3 +39,102 @@ export const DiseaseOutbreakCodes = {
+        ),
+        RTSL_ZEB_TEA_INCIDENT_STATUS: diseaseOutbreak.incidentStatus,
+        RTSL_ZEB_TEA_DATE_EMERGED: diseaseOutbreak.emerged.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_EMERGED_NARRATIVE: diseaseOutbreak.emerged.narrative,
+        RTSL_ZEB_TEA_DATE_DETECTED: diseaseOutbreak.detected.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_DETECTED_NARRATIVE: diseaseOutbreak.detected.narrative,
+        RTSL_ZEB_TEA_DATE_NOTIFIED: diseaseOutbreak.notified.date.toISOString(),
+        RTSL_ZEB_TEA_DATE_NOTIFIED_NARRATIVE: diseaseOutbreak.notified.narrative,
+        RTSL_ZEB_TEA_INITIATE_INVESTIGATION:
+            diseaseOutbreak.earlyResponseActions.initiateInvestigation.toISOString(),
+        RTSL_ZEB_TEA_CONDUCT_EPIDEMIOLOGICAL_ANALYSIS:
+            diseaseOutbreak.earlyResponseActions.conductEpidemiologicalAnalysis.toISOString(),
+        RTSL_ZEB_TEA_LABORATORY_CONFIRMATION: diseaseOutbreak.earlyResponseActions
+            .laboratoryConfirmation.na
+            ? "true"
+            : "",
+        RTSL_ZEB_TEA_SPECIFY_DATE1:
+            diseaseOutbreak.earlyResponseActions.laboratoryConfirmation.date?.toISOString() ?? "",
+        RTSL_ZEB_TEA_APPROPRIATE_CASE_MANAGEMENT: diseaseOutbreak.earlyResponseActions
+            .appropriateCaseManagement.na
+            ? "true"
+            : "",
+        RTSL_ZEB_TEA_SPECIFY_DATE2:
+            diseaseOutbreak.earlyResponseActions.appropriateCaseManagement.date?.toISOString() ??
+            "",
+        RTSL_ZEB_TEA_APPROPRIATE_PUBLIC_HEALTH: diseaseOutbreak.earlyResponseActions
+            .initiatePublicHealthCounterMeasures.na
+            ? "true"
+            : "",
+        RTSL_ZEB_TEA_SPECIFY_DATE3:
+            diseaseOutbreak.earlyResponseActions.initiatePublicHealthCounterMeasures.date?.toISOString() ??
+            "",
+        RTSL_ZEB_TEA_APPROPRIATE_RISK_COMMUNICATION: diseaseOutbreak.earlyResponseActions
+            .initiateRiskCommunication.na
+            ? "true"
+            : "",
+        RTSL_ZEB_TEA_SPECIFY_DATE4:
+            diseaseOutbreak.earlyResponseActions.initiateRiskCommunication.date?.toISOString() ??
+            "",
+        RTSL_ZEB_TEA_ESTABLISH_COORDINATION_MECHANISM:
+            diseaseOutbreak.earlyResponseActions.establishCoordination.toISOString(),
+        RTSL_ZEB_TEA_RESPONSE_NARRATIVE: diseaseOutbreak.earlyResponseActions.responseNarrative,
+        RTSL_ZEB_TEA_ASSIGN_INCIDENT_MANAGER: diseaseOutbreak.incidentManagerName,
+        RTSL_ZEB_TEA_NOTES: diseaseOutbreak.notes ?? "",
+    };
+}
+
+function getHazardTypeCode(hazardType: HazardType): string {
+    switch (hazardType) {
+        case "Biological:Animal":
+            return "BIOLOGICAL_ANIMAL";
+        case "Biological:Human":
+            return "BIOLOGICAL_HUMAN";
+        case "Chemical":
+            return "CHEMICAL";
+        case "Environmental":
+            return "ENVIRONMENTAL";
+        case "Unknown":
+            return "UNKNOWN";
+    }
+}
```

Comment:
Not that it's wrong, but note that a simple object `Record<HazardType, string>`.